osquery/logger/CMakeLists.txt

```diff
@@ -61,3 +61,12 @@ if(NOT SKIP_AWS AND NOT WINDOWS)
   ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-sts")
   ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-core")
 endif()
+
+if(NOT WINDOWS)
```

Comment:
Something in this CMakeLists file also has to include the `kafka_producer` source, am I missing something here?